README.md

```diff
@@ -1,86 +1,28 @@
-401 JS --  Lab 13 express with mongo
-===
+## Description
```

Comment:
Your best README so far. :1st_place_medal: :)

As a nitpick, it might also be good to tell a reader to ``brew install httpie`` so they can do the ``http`` commands in their terminal as that's not built-in. 

Great job!